src/app/components/PortraitVideoCarousel/PortraitVideoCarouselNavigation/index.tsx

```diff
@@ -15,6 +11,12 @@ const DEFAULT_TRANSLATION = {
   next: 'Scroll to next item',
 };
 
+type ScrollDirection = 'left' | 'right' | 'up' | 'down';
```

Comment:
why are 'up' and 'down available types for this scroll? Is it because the same types are used for vertical scroll in the video modal? Just because I haven't seen there being an up and down scroll used for video carousels.

Reply:
Not sure, it was there when I got here 😅 

I'll remove them though, can't see it being used anywhere and can be readded if needed.